report/src/main/webapp/app/app.module.js

```diff
@@ -69,6 +76,8 @@ define(['angularAMD',
   'unsavedChangesModalController'], function (angularAMD, _) {
 
   var app = angular.module('app', ['ui.router', 'ui.bootstrap']);
+  // jshint ignore:line
```

Comment:
Do we still need this comment for jshint here?

Reply:
No, I totally forgot about that one.

Reply:
Master branch was already merged into this branch ealier. Is something wrong with it?